### PR TITLE
feat: [small] add files for command functional test

### PR DESCRIPTION
**Pull request by s-yoshika**
This PR add some files for functional test of sd-cmd.

### CODE REVIEW

File: binary-command.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace: screwdriver-cd-test
+name: binary-test
+description: |
+   Test command for binary format
+maintainer: foo@bar.com
```
Comment (wdstar) [inline binary-command.yaml]: We should use `@example.com`

Comment (s-yoshika) [reply binary-command.yaml]: Fixed

**Review by wdstar — APPROVED** [review]

File: screwdriver.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+shared:
+    image: golang:latest
```
Comment (tk3fftk) [inline screwdriver.yaml]: [small comment] Should Indent be 2 spaces same as other lines?

**Review by tk3fftk — APPROVED** [review]
Otherwise LGTM :+1:

File: screwdriver.yaml (hunk 2 above)
Comment (s-yoshika) [reply screwdriver.yaml]: Thanks, I'll fix it.

**Pull request merged**